tests/helper/helper_oc.go

```diff
@@ -450,10 +450,31 @@ func (oc *OcRunner) WaitForDCRollout(dcName string, project string, timeout time
 	session.Wait(timeout)
 }
 
-// CheckForExistence checks if the given resource type exists on the cluster
-func (oc *OcRunner) CheckForExistence(resourceType, namespace string) {
+// WaitAndCheckForExistence wait for the given and checks if the given resource type gets deleted on the cluster
+func (oc *OcRunner) WaitAndCheckForExistence(resourceType, namespace string, timeoutMinutes int) bool {
```

Comment:
whats the point of adding it, if we are having 
https://github.com/openshift/odo/pull/2732, I tested your previous commit it was working well, the test was failing, it would get passed once https://github.com/openshift/odo/pull/2732 this will merge.

Reply:
To test that the resources are deleted even when no wait flag is passed

Reply:
Got, it